src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -694,244 +728,145 @@ const CursorGuide = styled('div')`
 
   transform: translateX(-50%);
 
-  height: ${SPAN_ROW_HEIGHT}px;
+  height: 100%;
 `;
 
 export const DividerLine = styled('div')`
   position: absolute;
-  height: ${SPAN_ROW_HEIGHT}px;
-
+  height: 100%;
+  width: 1px;
   transform: translateX(-50%);
 
+  /* increase hit target */
+  border-width: 0 5px;
+  border-color: rgba(0, 0, 0, 0);
+  border-style: solid;
+  box-sizing: content-box;
+  background-clip: content-box;
+
   background-color: #cdc7d5;
   z-index: ${zIndex.dividerLine};
 
   &.hovering {
-    width: 4px !important;
+    width: 3px;
     cursor: col-resize;
   }
-
-  ${({hovering}: {hovering: boolean}) => {
-    if (!hovering) {
-      return 'width: 2px;';
-    }
-
-    return `
-      width: 4px;
-      cursor: col-resize;
-      `;
-  }};
 `;
 
 const SpanBarTitleContainer = styled('div')`
   display: flex;
   align-items: center;
-
-  height: ${SPAN_ROW_HEIGHT}px;
+  height: 100%;
   position: absolute;
   left: 0;
   top: 0;
   width: 100%;
+  user-select: none;
 `;
 
 const SpanBarTitle = styled('div')`
   position: relative;
-  top: 0;
-
-  height: ${SPAN_ROW_HEIGHT}px;
-  line-height: ${SPAN_ROW_HEIGHT}px;
-
-  color: #4a3e56;
-  font-size: 12px;
-
-  user-select: none;
-
+  height: 100%;
+  color: ${p => p.theme.gray4};
+  font-size: ${p => p.theme.fontSizeSmall};
   white-space: nowrap;
-
   display: flex;
+  flex: 1;
   align-items: center;
 `;
 
 const SpanTreeTogglerContainer = styled('div')`
   position: relative;
-  top: 0;
-
   height: 15px;
-
-  max-width: 40px;
   width: 40px;
-  min-width: 40px;
-
-  margin-right: 8px;
-
+  min-width: 40px; /* annoying flex thing */
+  margin-right: ${space(1)};
   z-index: ${zIndex.spanTreeToggler};
-
-  user-select: none;
-
   display: flex;
   justify-content: flex-end;
 `;
 
-const SpanTreeToggler = styled('div')`
-  position: relative;
+const getTogglerTheme = ({isExpanded}) => {
+  return isExpanded ? theme.button.default : theme.button.primary;
+};
 
-  white-space: nowrap;
+const getTogglerHoverTheme = ({isExpanded}) => {
+  return getTogglerTheme({isExpanded: !isExpanded});
+};
 
+const SpanTreeToggler = styled.div<{isExpanded: boolean}>`
+  white-space: nowrap;
   height: 15px;
   min-width: 25px;
-
-  padding-left: 4px;
-  padding-right: 4px;
-
+  padding: 0 4px;
   display: flex;
-  flex-wrap: nowrap;
   align-items: center;
-  align-content: center;
   justify-content: center;
-
-  > span {
-    flex-grow: 999;
-  }
-
+  border-radius: 99px;
   transition: all 0.15s ease-in-out;
+  font-size: ${p => p.theme.fontSizeExtraSmall};
+  line-height: 0;
 
-  border-radius: 99px;
+  background: ${p => getTogglerTheme(p).background};
```

Comment:
I wonder if it'd be easier to read if `getTogglerTheme` returns a string of `background`, `border`, and `color`.

Reply:
you mean like `[String, String, String]`?

Reply:
Nope, like the actual CSS to be used as a string.

Reply:
ah yeah—I considered this, but let me give it a deeper look.